lib/DDG/Spice/EconomicIndicators.pm

```diff
@@ -0,0 +1,93 @@
+package DDG::Spice::EconomicIndicators;
+# ABSTRACT: 
+# Instant answer for economic indicators for different countries
+# The indicators considered are
+#   a) Gross Domestic Product
+#   b) Annual Growth Rate
+#   c) Per Capita Income
+# All data used in this instant answer comes form World Bank(http://data.worldbank.org)
+
+use DDG::Spice;
+use Time::Piece;
+use JSON;
+
+spice is_cached => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "EconomicIndicators";
+source "http://data.worldbank.org";
+description "Gives information about economic indicators of a country( Gross Domestic Product, Per Capita Income, Growth Rate)";
+primary_example_queries "gdp of india", "china per capita income","india growth rate";
+category "finance";
+topics "economy_and_finance";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/EconomicIndicators.pm";
+attribution github => ["gauravtiwari5050", "Gaurav Tiwari"];
+
+# Triggers
+triggers any => "gdp", "gross domestic product","growth rate","per capita income";
+
+#the world bank api requires date range in years
+#we will use the current year, and three previous years
+spice to => 'http://api.worldbank.org/countries/$1?per_page=10&date='.localtime->add_years(-3)->year.':'.localtime->year.'&format=json';
```

Comment:
For calling the worldbank apis we need a date range, for which the current system time would suffice. We dont neccessarily need the location of the user for this purpose.